### PR TITLE
Fixing C# Basics Workshop

### PR DESCRIPTION
**Pull request by webreidi**
updating to be more accurate

### CODE REVIEW

File: content/english/csharp-basics/_index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The C# programming language was designed by Anders Hejlsberg from Microsoft in 2
 A high-level programming language is called high-level because it is quite a few steps away from the original code run on a computer's Central Processing Unit (CPU). High-level code is meant to be used by humans and is much easier to understand than low-level code. The high-level code is later translated into a low-level language, which can only be perceived and run by a specific CPU. For the code to work, every single line of code written in a high-level language must be translated into machine language before being implemented by the computer.
 {{% /notice %}}
 
-C# is one of the most popular programming languages in the world, along with Java, JavaScript, Python and others. C# runs on Windows and various Linux operating systems.
+C# is one of the most popular programming languages in the world, along with Java, JavaScript, Python and others. C# can run on any computer, including on Windows and various Linux operating systems.
```
[the remark this answers is not in the document]
Comment (sunnynagavo) [reply content/english/csharp-basics/_index.md:21]: Looks good, resolved

File: content/english/csharp-basics/operators.md [hunk 2]
```diff
@@ -70,7 +71,7 @@ The following are the relational operators in C#:
 `<=` | less than or equal to | `(2 <= 2)` is `true`
 
 Note: `==` and `!=` can only be used on data of the same type. For example:
-```java
+```csharp
```
Comment (sunnynagavo) [inline content/english/csharp-basics/operators.md:74]: Good catch

**Review by sunnynagavo — APPROVED** [review]
LGTM.

**Review by ozhang22 — APPROVED** [review]

File: content/english/csharp-basics/write-to-console.md [hunk 3]
```diff
@@ -15,14 +15,6 @@ Launch the repl and fork it. Give it a try by pressing the **run** button.  The
...
 
-Note that all your code must be between the two braces:
-
-```c#
-public static void Main (string[] args) {
-     //your code here
-}
-```
-
 ### Let's write come code!
```
Comment (ozhang22) [inline content/english/csharp-basics/write-to-console.md:18]:
```suggestion
### Let's write some code!
```

File: content/english/csharp-basics/write-to-console.md
Comment (ozhang22) [inline content/english/csharp-basics/write-to-console.md:1]: On line 32, does \`\`\`c# work? I think you need to change it to \`\`\`csharp

File: content/english/csharp-basics/classes-and-objects.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+date: 2024-09-16T00:00:00Z
 weight: 8
 ---
 
 ## What is an Object and what is a Class?
 
-An object is an thing (noun) that has certain characteristics and can perform certain tasks. A Class is the blueprint or definition for that object when it is created.
+An Object is a thing (noun) that has certain characteristics and can perform certain tasks. A Class is the blueprint or definition for that object when it is created.
 
 An example is a Person. A Person is a kind of Class. A person can have certain characteristics that distinguish it from another person. It can have blue eyes, it can be 15 years old, etc. A person can also perform certain tasks. It can walk, it can talk, etc.
```
Comment (ozhang22) [inline content/english/csharp-basics/classes-and-objects.md:13]: These are methods that "Bea"

There's an extra space

File: content/english/csharp-basics/methods.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Methods"
 description: "Introduce methods in C#."
-date: 2023-09-13T00:00:00Z
+date: 2024-09-16T00:00:00Z
 weight: 7
 ---
```
Comment (ozhang22) [inline content/english/csharp-basics/methods.md:7]: Can you format if, for and while on line 10?

File: content/english/csharp-basics/operators.md [hunk 6]
```diff
@@ -37,11 +37,12 @@ The following are the arithmetic operators in C#:
...
 **Note**: If both of the operands of division are integers, the result will also be an integer. For example, `10 / 4` returns 2, not 2.5 since we throw away the remainder. If either of the operands is a double, the result will be a double.
 
-**Note**: Modulus operation get the remainder of the division operation.
+**Note**: Use the modulus operation (`%`) get the remainder of the division operation.
+
 {{% /notice %}}
 
 {{% notice tip %}}
-Add (`+`) is used on String as concatenation operator. For example:
+Add (`+`) operator is used on Strings as a concatenation operator. For example:
```
Comment (ozhang22) [inline content/english/csharp-basics/operators.md:45]:
```suggestion
The add (`+`) operator is used on Strings as a concatenation operator. For example:
```

**Review by ozhang22 — APPROVED** [review]
Looks good! Just some small changes

**Pull request merged**